workshops/templates/workshops/person_bulk_add_results.html

```diff
@@ -25,13 +25,13 @@
                 <input type="text" name="personal" value="{{ entry.personal|default:"" }}" class="hidden">
             </td>
             <td class="editable">
-                <span>{{ entry.middle|default:"&mdash;" }}</span>
-                <input type="text" name="middle" value="{{ entry.middle|default:"" }}" class="hidden">
-            </td>
-            <td class="editable">
                 <span>{{ entry.family|default:"&mdash;" }}</span>
                 <input type="text" name="family" value="{{ entry.family|default:"" }}" class="hidden">
             </td>
+            <td {% if not entry.person_exists %}class="editable"{% endif %}>
+                <span>{{ entry.username|default:"&mdash;" }}</span>
+                <input type="text" name="username" value="{{ entry.username|default:"" }}" class="hidden">
+            </td>
```

Comment:
So when I do bulk upload, is the editable table pre-populated with machine-generated usernames? If so, where is that done?